master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -59,23 +59,23 @@ kubectl expose --namespace=advanced-policy-demo deployment nginx --port=80
 > For example: `export ETCD_ENDPOINTS=http://10.96.232.136:6666`.
 {: .alert .alert-info}
 
-Now that we've created a Namespace and a set of pods, we should see those objects show up in the
+Now that we've created a namespace and a set of pods, we should see those objects show up in the
 Calico API using `calicoctl`.
 
-We can see that the Namespace has a corresponding [Profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile).
+We can see that the namespace has a corresponding [profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile).
 
 ```shell
 $ calicoctl get profile -o wide
 NAME                          TAGS
```

Comment:
Oh @tmjd one more thing I wanted to mention and maybe you are already aware of this but we still see `TAGS` in the heading. Maybe that heading still shows up in the output and just has nothing beneath it...I am not sure. Just wanted to make sure you were aware of this detail.

Reply:
Thank you for checking but yeah that is how it looks with the `TAGS` heading and nothing in that column.